migrations/20260724202500-retire-release-bus-v1-tables.js

```
@@ -0,0 +1,55 @@
+      .map(function (pair) {
+        return '`' + pair[0] + '` TO `' + pair[1] + '`';
+      })
+      .join(', ')
+  );
+}
+
+exports.up = function (db) {
+  return db.runSql(renameSql(TABLE_RENAMES));
+};
+
+exports.down = function (db) {
```

ESLint

[error] 'exports' is not defined.

(no-undef)